packages/playground/src/package.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
there should be a single package json in `packages/playground` (so this one in `src` should be extracted one level up.